code/controllers/configuration/entries/general.dm

```diff
@@ -202,6 +202,9 @@
 
 /datum/config_entry/string/banappeals
 
+/datum/config_entry/string/siteurl
+	config_entry_value = "https://beestation13.com"
+
 /datum/config_entry/string/wikiurl
 	config_entry_value = "http://www.tgstation13.org/wiki"
```

Comment:
Should probably update all of the old TG defaults.

Reply:
Mmm, good point.